### ISSUE TITLE
runc not installed in Ubuntu install process

### ISSUE THREAD

**Issue by anarcat** [projectatomic/buildah#355]
After following the Ubuntu instructions (or, really, the modified instructions for Debian in #354), I have found that I cannot actually build containers that require the `RUN` command:

```
subsonic-docker-image1$ sudo buildah bud -t anarcat/debian-airsonic .
STEP 1: FROM debian:stretch
Getting image source signatures
Copying blob sha256:3e17c6eae66cd23c59751c8d8f5eaf7044e0611dc5cebb12b1273be07cdac242
 43.04 MiB / 43.04 MiB [===================================================] 16s
Copying config sha256:6d83de432e98210aa25bcc5556a641d60ec621b67786a2862cfeec5d7258a4d0
 1.47 KiB / 1.47 KiB [======================================================] 0s
Writing manifest to image destination
Storing signatures
STEP 2: MAINTAINER michael@schuerig.de
STEP 3: ENV DEBIAN_FRONTEND noninteractive
STEP 4: ENV SONIC_USER airsonic
STEP 5: ENV SONIC_DIR /var/airsonic
STEP 6: ENV WAR_URL https://github.com/airsonic/airsonic/releases/download/v10.0.0/airsonic.war
STEP 7: RUN useradd --home "$SONIC_DIR" -M "$SONIC_USER" -K UID_MIN=10000 -K GID_MIN=10000 &&     mkdir -p "$SONIC_DIR" && chown "$SONIC_USER" "$SONIC_DIR" && chmod 0770 "$SONIC_DIR"
error building: error building at step {Env:[WAR_URL=https://github.com/airsonic/airsonic/releases/download/v10.0.0/airsonic.war SONIC_DIR=/var/airsonic SONIC_USER=airsonic DEBIAN_FRONTEND=noninteractive PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin     DEBIAN_FRONTEND=noninteractive SONIC_USER=airsonic SONIC_DIR=/var/airsonic WAR_URL=https://github.com/airsonic/airsonic/releases/download/v10.0.0/airsonic.war] Command:run Args:[useradd --home "$SONIC_DIR" -M "$SONIC_USER" -K UID_MIN=10000 -K GID_MIN=10000 &&     mkdir -p "$SONIC_DIR" && chown "$SONIC_USER" "$SONIC_DIR" && chmod 0770 "$SONIC_DIR"] Flags:[] Attrs:map[] Message:RUN useradd --home "$SONIC_DIR" -M "$SONIC_USER" -K UID_MIN=10000 -K GID_MIN=10000 &&     mkdir -p "$SONIC_DIR" && chown "$SONIC_USER" "$SONIC_DIR" && chmod 0770 "$SONIC_DIR" Original:RUN useradd --home "$SONIC_DIR" -M "$SONIC_USER" -K UID_MIN=10000 -K GID_MIN=10000 &&     mkdir -p "$SONIC_DIR" && chown "$SONIC_USER" "$SONIC_DIR" && chmod 0770 "$SONIC_DIR"}: exec: "runc": executable file not found in $PATH
```

The Dockerfile is available in [this repository](https://github.com/anarcat/subsonic-docker-image/tree/airsonic) but I doubt it is Dockerfile-specific: obviously, the install process I followed does not install `runc` at all, even though it is documented as a dependency.

I did notice there's a `runc` symlink after the build, and that `make` somehow checks out a copy of [runc](https://github.com/opencontainers/runc) in the GOPATH *and* builds it, but somehow it doesn't actually gets installed in `/usr/local/bin` when we run `make install` in buildah.

What I ended up doing was:

```
cd ../../opencontainers/runc
install -m 0755 runc /usr/local/bin/
```

Shouldn't that be done as part of the build process? or should we instruct users to go get runc? or make install in runc?

**Comment by anarcat** [comment]
okay, i'll figure something out in #354 for debian at least then...

**Comment by anarcat** [comment]
fix proposed in af8aa13 as part of #354.
